### PR TITLE
refactor(components): [scrollbar] switch to script-setup syntax

### PR DESCRIPTION
**Pull request by holazz**
https://github.com/element-plus/element-plus/issues/6114

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by holazz** [comment]
cc @sxzz

File: packages/components/scrollbar/src/scrollbar.ts [hunk 1]
```diff
@@ -10,10 +11,7 @@ export const scrollbarProps = buildProps({
     type: [String, Number],
     default: '',
   },
-  native: {
-    type: Boolean,
-    default: false,
-  },
+  native: Boolean,
```
Comment (sxzz) [inline packages/components/scrollbar/src/scrollbar.ts:14]: Why did you change it? I think we can keep it as-is.

File: packages/components/scrollbar/src/scrollbar.ts [hunk 2]
```diff
@@ -35,16 +33,12 @@ export const scrollbarProps = buildProps({
     type: String,
     default: 'div',
   },
-  always: {
-    type: Boolean,
-    default: false,
-  },
+  always: Boolean,
```
Comment (sxzz) [inline packages/components/scrollbar/src/scrollbar.ts:36]: Same here

File: packages/components/scrollbar/src/scrollbar.ts (hunk 1 above)
Comment (holazz) [reply packages/components/scrollbar/src/scrollbar.ts:14]: I think the default value of Boolean props is `false`, so I simplified the code. 
refer to: https://github.com/vuejs/core/issues/5863#issuecomment-1118446966

**Review by sxzz — APPROVED** [review]

**Comment by sxzz** [comment]
https://github.com/element-plus/element-plus/runs/6662396264?check_suite_focus=true

Could you please check this?

**Comment by holazz** [comment]
> [element-plus/element-plus/runs/6662396264?check_suite_focus=true](https://github.com/element-plus/element-plus/runs/6662396264?check_suite_focus=true)
> 
> Could you please check this?
> 
> **Update: never mind**

I'm debugging, it seems to be caused by removing `scrollbar` from `type-unsafe`, I don't need to care about this now?

**Comment by holazz** [comment]
> I found that the table header does not scroll with the body, maybe the Scrollbar components should expose more values.
> 
> eq: scrollbar, resize$ ...

I found that `wrap$` is used in `table`, I fixed it here https://github.com/element-plus/element-plus/pull/8035
